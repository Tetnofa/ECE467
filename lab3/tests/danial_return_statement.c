int main() {
    return true;
}
int f() {
    return;
}
void g() {
    return;
}
void gf() {
    return 2;
}
bool aldkaw(int x) {
    return x;
}
int a (int x, int y, int z) {
    {
        {
            {
                {
                    return z;
                }
            }
        }
    }
}
int gfcnad(int arr[]) {
    return arr[2];
}
void adlkdjwad(int b) {
    int f;
    return f;
}
bool adlkjawd(int f) {
    return f;
}
bool dawjdhaw(bool g[]) {
    return g[2];
}
int daawdawd(bool g[]) {
    return g[2];
}
void adwaawdjawdd(int x) {
    return x;
}