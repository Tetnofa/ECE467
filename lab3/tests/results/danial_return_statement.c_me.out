sema: 2:4 : mismatched return statement 
sema: 5:4 : mismatched return statement 
sema: 11:4 : mismatched return statement 
sema: 14:4 : mismatched return statement 
sema: 32:4 : mismatched return statement 
sema: 35:4 : mismatched return statement 
sema: 41:4 : mismatched return statement 
sema: 44:4 : mismatched return statement 
